src/chrome/content/rules/thecustomizewindows.net.xml

```diff
@@ -0,0 +1,10 @@
+<ruleset name="TheCustomizeWindows.net">
+	<target host="thecustomizewindows.net" />
+	<target host="www.thecustomizewindows.net" />
+
+	<securecookie host=".+" name=".+"/>
+
+	<rule from="^http://(?:www\.)?thecustomizewindows\.net/"
```

Comment:
Please replace non-capturin group (`(?:www\.)?`) with `(www\.)?`.
Could you also add a comment why this redirect for .net to .com is needed and that .com is already preloaded? This makes it easier for future maintenance.